sync3/dispatcher.go

```diff
@@ -141,24 +143,15 @@ func (d *Dispatcher) OnNewInitialRoomState(ctx context.Context, roomID string, s
 	}
 }
 
-// Called by v2 pollers when we receive new events
-func (d *Dispatcher) OnNewEvents(
```

Comment:
Are we just getting ride of this for ergonomics' sake? I.e. was there something buggy about this?

Perhaps the point is that we want to force the caller to thinking about handling events with different `pos`es?

Reply:
Yes, it was originally useful for ergonomics to avoid needing callers to for .. range it. However, in practice we would need to for .. range it anyway to link the event NID and the JSON, so we might as well just remove the bulk function.